src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -12,6 +12,18 @@ body {
     cursor: pointer;
 }
 
+/*
+Apply to alert to override Bootstrap's default 3d alerts
+*/
+.alert {
+    -webkit-box-shadow: none;
+    box-shadow: none;
+}
+
+.alert-warning, .alert-success, .alert-info, .alert-danger {
```

Comment:
I just noticed that our [CSS coding standard](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub) specifies that selectors must be written in separate lines. While there are other violations of this in the file, let's not introduce more. I'll open another issue for that.